src/main/pg/rx.h

```diff
@@ -62,6 +62,7 @@ typedef struct rxConfig_s {
     uint8_t srxl2_baud_fast;                   // Select Spektrum SRXL2 fast baud rate
     uint8_t sbus_baud_fast;                    // Select SBus fast baud rate
     uint32_t msp_override_channels_mask;       // Channels to override when the MSP override mode is enabled
+    uint8_t msp_override_failsafe;             // if true then extra RC link is always required in msp_override mode, false - allows control via msp_override without extra RC link (autonomy use case)
```

Comment:
```suggestion
    uint8_t msp_override_failsafe;             // if false then extra RC link is always required in msp_override mode, true - allows control via msp_override without extra RC link (autonomous use case)
```

Reply:
Didn't check but if OFF is the default that is good, as we need to enable it for autonomous use explicitly.